src/backend/main.ts

```diff
@@ -1144,15 +1144,20 @@ ipcMain.on('logError', (e, err) => logError(err, LogPrefix.Frontend))
 ipcMain.on('logInfo', (e, info) => logInfo(info, LogPrefix.Frontend))
 
 let powerDisplayId: number | null
-const gamePlaySessionStartTimes: Record<string, bigint> = {}
+let gamePlaySessionStartTimes: Record<string, bigint> = {}
 
 function startNewPlaySession(appName: string) {
+  gamePlaySessionStartTimes = {}
```

Comment:
I see, you just clear the object when another game is launched. Why not just make it a single gamePlaySessionStartTime?